packages/xo-web/src/common/action-button.js

```diff
@@ -50,7 +52,7 @@ export default class ActionButton extends Component {
     //
     // if a function, it will be called with the result of the action to
     // compute the path
-    redirectOnSuccess: PropTypes.oneOfType([PropTypes.func, PropTypes.string]),
+    redirectOnSuccess: PropTypes.oneOfType([PropTypes.func, PropTypes.string, PropTypes.number]),
```

Comment:
You can use `PropTypes.oneOf([this.GO_BACK])` for a more specific check.

I believe it composes with `oneOf` but I might be wrong.